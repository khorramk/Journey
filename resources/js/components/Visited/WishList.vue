<template>
    <div class="">
        <Nav></Nav>
        <div class="container-for-WishList" v-if="this.$store.state.Modal.open === false">

            <Avatar/>
            <Search/>
            <AddButton/>
            <div class="container">
                <li v-for="(wish, i) in wishList" :key="i">
                    <div class="card">
                        {{i}}
                    </div>
                </li>
            </div>
        </div>
         <Modal v-if="this.$store.state.Modal.open && this.$store.state.Modal.close === false"/>
    </div>
</template>
<script>
import Avatar from '../dashboard/Avatar';
import Search from '../search/Search';
import AddButton from '../Visited/AddButton';
import Modal from '../Modal';
import Nav from '../Nav';
    export default {
        data() {
            return {
                wishList: Array(30).fill(1,9)
            }
        },
        components: {
            Nav,
            Avatar,
            Search,
            AddButton,
            Modal

        },
    }
</script>

<style lang="scss" scoped>

</style>
