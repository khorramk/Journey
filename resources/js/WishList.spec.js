import { mount } from '@vue/test-utils';
import Vue from 'vue';
import axios from 'axios';
import VueRouter from 'vue-router';
import WishList from './components/Visited/WishList';


const vm = new Vue(WishList);

it('renders the elements', () => {

    vm.$nextTick().then(()=> {

     expect(vm).toMatchSnapshot();
    }).catch().finally()
  })
