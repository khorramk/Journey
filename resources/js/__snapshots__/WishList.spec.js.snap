// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the elements 1`] = `
Vue {
  "$attrs": Object {},
  "$children": Array [],
  "$createElement": [Function],
  "$listeners": Object {},
  "$options": Object {
    "_base": [Function],
    "components": Object {
      "AddButton": Object {
        "methods": Object {
          "showFullModal": [Function],
        },
        "render": [Function],
        "staticRenderFns": Array [],
      },
      "Avatar": Object {
        "data": [Function],
        "methods": Object {
          "insertAvatar": [Function],
          "onSelectFile": [Function],
          "openSelection": [Function],
          "removeSelection": [Function],
        },
        "render": [Function],
        "staticRenderFns": Array [],
      },
      "Nav": Object {
        "data": [Function],
        "methods": Object {
          "logout": [Function],
          "popLogOutBtn": [Function],
        },
        "render": [Function],
        "staticRenderFns": Array [],
      },
      "Search": Object {
        "data": [Function],
        "props": Object {
          "countryList": Object {
            "default": Array [],
            "type": [Function],
          },
        },
        "render": [Function],
        "staticRenderFns": Array [],
      },
      "WishListModal": Object {
        "computed": Object {
          "populate": [Function],
        },
        "data": [Function],
        "methods": Object {
          "inputslist": [Function],
          "onSelectFile": [Function],
          "submit": [Function],
        },
        "mounted": [Function],
        "render": [Function],
        "staticRenderFns": Array [],
      },
    },
    "data": [Function],
    "directives": Object {},
    "filters": Object {},
    "render": [Function],
    "staticRenderFns": Array [],
  },
  "$parent": undefined,
  "$refs": Object {},
  "$root": [Circular],
  "$scopedSlots": Object {},
  "$slots": Object {},
  "$vnode": undefined,
  "_c": [Function],
  "_data": Object {
    "wishList": Array [
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
    ],
  },
  "_directInactive": false,
  "_events": Object {},
  "_hasHookEvent": false,
  "_inactive": null,
  "_isBeingDestroyed": false,
  "_isDestroyed": false,
  "_isMounted": false,
  "_isVue": true,
  "_renderProxy": Vue {
    "$attrs": Object {},
    "$children": Array [],
    "$createElement": [Function],
    "$listeners": Object {},
    "$options": Object {
      "_base": [Function],
      "components": Object {
        "AddButton": Object {
          "methods": Object {
            "showFullModal": [Function],
          },
          "render": [Function],
          "staticRenderFns": Array [],
        },
        "Avatar": Object {
          "data": [Function],
          "methods": Object {
            "insertAvatar": [Function],
            "onSelectFile": [Function],
            "openSelection": [Function],
            "removeSelection": [Function],
          },
          "render": [Function],
          "staticRenderFns": Array [],
        },
        "Nav": Object {
          "data": [Function],
          "methods": Object {
            "logout": [Function],
            "popLogOutBtn": [Function],
          },
          "render": [Function],
          "staticRenderFns": Array [],
        },
        "Search": Object {
          "data": [Function],
          "props": Object {
            "countryList": Object {
              "default": Array [],
              "type": [Function],
            },
          },
          "render": [Function],
          "staticRenderFns": Array [],
        },
        "WishListModal": Object {
          "computed": Object {
            "populate": [Function],
          },
          "data": [Function],
          "methods": Object {
            "inputslist": [Function],
            "onSelectFile": [Function],
            "submit": [Function],
          },
          "mounted": [Function],
          "render": [Function],
          "staticRenderFns": Array [],
        },
      },
      "data": [Function],
      "directives": Object {},
      "filters": Object {},
      "render": [Function],
      "staticRenderFns": Array [],
    },
    "$parent": undefined,
    "$refs": Object {},
    "$root": [Circular],
    "$scopedSlots": Object {},
    "$slots": Object {},
    "$vnode": undefined,
    "_c": [Function],
    "_data": Object {
      "wishList": Array [
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
        1,
      ],
    },
    "_directInactive": false,
    "_events": Object {},
    "_hasHookEvent": false,
    "_inactive": null,
    "_isBeingDestroyed": false,
    "_isDestroyed": false,
    "_isMounted": false,
    "_isVue": true,
    "_renderProxy": [Circular],
    "_self": [Circular],
    "_staticTrees": null,
    "_uid": 0,
    "_vnode": null,
    "_watcher": null,
    "_watchers": Array [],
    "wishList": Array [
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
      1,
    ],
  },
  "_self": [Circular],
  "_staticTrees": null,
  "_uid": 0,
  "_vnode": null,
  "_watcher": null,
  "_watchers": Array [],
  "wishList": Array [
    undefined,
    undefined,
    undefined,
    undefined,
    undefined,
    undefined,
    undefined,
    undefined,
    undefined,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
    1,
  ],
}
`;
